### PR TITLE
[WFCORE-3136] Added Generic type CLI commands into docs (CLI guide)

### PR DESCRIPTION
**Pull request by Dvikasred**
Fix for issue, https://issues.jboss.org/browse/WFCORE-3136
Downstream issue: https://issues.jboss.org/browse/JBEAP-5042

### CODE REVIEW

**Comment by bstansberry** [comment]
this is ok to test

File: cli/src/main/resources/help/command.txt [hunk 1]
```diff
@@ -29,7 +29,7 @@ DESCRIPTION
   Here is an example of invoking operation 'flush-all-connection-in-pool' on
   datasource with jndi-name 'myds'.
 
-  data-source flush-all-connection-in-pool --jndi-name=myds
+  data-source flush-all-connection-in-pool --jndi-name=myds
```
Comment (jfdenise) [inline cli/src/main/resources/help/command.txt]: This ending char should be removed.

File: cli/src/main/resources/help/command.txt [hunk 2]
```diff
@@ -46,7 +46,13 @@ DESCRIPTION
 
   data-source --jndi-name=my-new-ds --min-pool-size=11 --max-pool-size=22
 
-  Tab-completion will help completing operation and property names.
+  
+ARGUMENTS
+  
+  Arguments should be passed with the commands, in CLI mode command we are having arguments(--help, --help --commands and etc)
+
+  For example, command /subsystem=datasources/data-source.
```
Comment (jfdenise) [inline cli/src/main/resources/help/command.txt]: Same, do we really need this sentence? The command action (add/remove/list) is missing. Perhaps example should be moved in the "Managing Commands"

File: cli/src/main/resources/help/command.txt [hunk 3]
```diff
@@ -46,7 +46,13 @@ DESCRIPTION
...
+  Tab-completion will help completing operation, property names and arguments.
```
Comment (jfdenise) [inline cli/src/main/resources/help/command.txt]: Do we have completion there for arguments? Is it working? Completion doesn't seem to work for argument value.

File: cli/src/main/resources/help/command.txt [hunk 4]
```diff
@@ -46,7 +46,13 @@ DESCRIPTION
 
   data-source --jndi-name=my-new-ds --min-pool-size=11 --max-pool-size=22
 
-  Tab-completion will help completing operation and property names.
+  
+ARGUMENTS
+  
+  Arguments should be passed with the commands, in CLI mode command we are having arguments(--help, --help --commands and etc)
```
Comment (jfdenise) [inline cli/src/main/resources/help/command.txt]: Do we really need this sentence?

File: cli/src/main/resources/help/command.txt [hunk 5]
```diff
@@ -46,7 +46,13 @@ DESCRIPTION
 
   data-source --jndi-name=my-new-ds --min-pool-size=11 --max-pool-size=22
 
-  Tab-completion will help completing operation and property names.
+  
+ARGUMENTS
```
Comment (jfdenise) [inline cli/src/main/resources/help/command.txt]: Seems to me that the ARGUMENTS should contain what the "Managing Operations" contains. This document is the documentation of "command" not of the generated command.
Then the Managing Operation should be named "Generated command"

File: cli/src/main/resources/help/command.txt [hunk 6]
```diff
@@ -91,3 +100,18 @@ ARGUMENTS
             removed.
 			
   list    - lists existing generic type commands.
+ 
+
+  For example,Login to CLI mode and run the below commands,
```
Comment (jfdenise) [inline cli/src/main/resources/help/command.txt]: Perhaps an EXAMPLE chapter containing the list/add/remove examples? Do we need "Login to CLI mode and run"?

**Comment by jfdenise** [comment]
@Dvikasred, this one never progressed. Do you plan to make updated to help finally? If not, then we can close it.

**Comment by jmesnil** [comment]
@Dvikasred I'm closing this PR as there has been no progress since @jfdenise requested changes in May.
Do not hesitate to reopen this PR if you plan to address it, thanks

**Pull request closed without merging**